### ISSUE TITLE
How to make Selecton work in text fields?

### ISSUE THREAD

**Issue by jupiter104** [emvaized/selecton-extension#24]
There seems to be a special panel for text fields in the options but why can't we have the regular panel?

**Comment by emvaized** [comment]
Because I can't think of a reason when regular buttons (search, translate etc.) can be really useful for text you've just inputted yourself. These cases seem to be quite rare, and since your hands already on keyboard you could always `CTRL+T` and quickly search the text you need – and in the meantime additional buttons will overflood the panel, already containing Cut, Copy and Paste buttons. 

That's why it was decided to separate textfield panel and regular panel.

**Comment by simonbcn** [comment]
I also vote for this feature. 
For example, I am living in Denmark (_for a few years now_) but I don't speak Danish, so when I want to buy something online, I translate the name of the object from my language to Danish and then select the translation to search for it in Google Denmark (_to restrict it to Danish online stores only_). I recognize that this is a very specific case. 

But also at other times there are websites that display text in text fields, either for aesthetics, page design, etc.

**Comment by emvaized** [comment]
Perhaps the Translate button is the only one somewhat justified to be displayed for text fields. However, from your description it should behave differently and replace the selected text with it's translation on click.

**Comment by simonbcn** [comment]
@emvaized no, you have misunderstood me. I am not talking about the translate button, but the search button. 
In addition to the above example there are others, for example, I am typing a word in a text field and I want to look it up to be sure it is spelled that way or its meaning.

**Comment by emvaized** [comment]
Maybe in future there will be an option for this – or a full-fledged button editor for both regular and text field tooltips. But for now I'm not planning to work in this direction.

**Comment by emvaized** [comment]
I came back to this idea after some time, and now it appears to me that it is not really necessary to add a full-featured button editor to the extension to implement it. It would probably be enough to add an option that simply displays “Search” and “Translate” buttons for text fields instead of the current text formatting buttons (Italics, Bold and Strikethrough), which are probably not very popular anyway. 

However, some ideas have occurred to me and I was wondering how exactly you would like to use these buttons in text fields? For example, if we're talking about a “Translate” button, would you like it to just open the translation in a new tab when clicked, like a button on a regular panel does? Or would it be better if it inserted the translation in place of the selected text on click? It appears to me that the second scenario might have more useful applications.

**Comment by FrankHGS** [comment]
嗯嗯! 我支持這項功能! 這在如 stable diffusion webui 上編寫提示詞顯得非常有用, 我可以移除許多類似的翻譯插件與擴充
非常感謝!!
